app/views/calculation/resident/outsideTaxYear.scala.html

```diff
@@ -41,6 +41,10 @@ <h1 class="govuk-heading-xl">@Messages("calc.resident.outsideTaxYears.title")</h
         <a id="change-date-link" class="back-link" href="@navBackLink">@Messages("calc.resident.outsideTaxYears.changeDate")</a>
     }
 
-    <p><a class="govuk-button" href="@continueUrl" id="submit">@Messages("calc.base.button.continue")</a></p>
+    <p>
+        <a class="govuk-button" href="@continueUrl" id="submit" role="button" data-module="govuk-button">
+            @Messages("calc.base.button.continue")
+        </a>
+    </p>
 
```

Comment:
I think it's better we use GovukButton component as below
@govukButton(Button(
                    href = continueUrl, 
	            content = Text(messages("calc.base.button.continue")),
	            attributes = Map("id" -> "submit")
	        ))

Reply:
Keeping as a link per recommendation in refinement